open-api/immich-openapi-specs.json

```diff
@@ -7519,6 +7558,10 @@
       },
       "CLIPConfig": {
         "properties": {
+          "duplicateThreshold": {
+            "format": "float",
+            "type": "number"
+          },
```

Comment:
Probably shouldn't be under CLIPConfig??? or is this still the case? Since you separated it from CLIP a little.

Reply:
Oh, I forgor to update the API